Toolset/palettes/inspector/editors/com.livecode.pi.gradientramp.behavior.livecodescript

```diff
@@ -87,6 +87,7 @@ on updateColorValue pColor
    lock screen
    lock messages
    set the swatchColor of widget "color" of me to pColor
+   put pColor into field "value" of me
```

Comment:
@BerndN This patch effectively makes `updateColorValue` and `updateColorDisplay` exactly the same. I have a tendency to think if there's two such similar implementations there must be a reason for the difference. I'm sure @livecodeali will know!